saleor/webhook/tests/test_webhook_payloads.py

```diff
@@ -722,8 +722,8 @@ def test_generate_invoice_payload(fulfilled_order):
             "token": str(invoice.order.id),
             "id": graphene.Node.to_global_id("Order", invoice.order.id),
             "language_code": "en",
-            "private_metadata": {},
-            "metadata": {},
+            "private_metadata": invoice.order.private_metadata,
```

Comment:
Is it expected? This should always be equal to `{}` right?

Reply:
This is a regular invoice payload for invoice webhooks, not a notification.